### PR TITLE
Page header

### PR DESCRIPTION
**Pull request by noahSerafin**
snapshots tests done

### CODE REVIEW

File: src/components/PageHeader/PageHeader.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import HeaderImageVideoContainer from "../../components/HeaderImageVideoContainer/HeaderImageVideoContainer";
+
+const PageHeader = (props) => {
+  const { title, subtitle, time, date, venue, location, content, isVideo } =
+    props;
+
+  const headerClass =
+    (time, date, venue, location, content, isVideo) == null
+      ? "page-header-only"
+      : "page-header";
```
Comment (DanForder) [inline src/components/PageHeader/PageHeader.jsx:14]: add a modifier class for --only:

```
let headerClass = 'page-header';
if(time, date, venue, location, content, isVideo) == null{
    headerClass += ' page-header--only';
}
```

File: src/pages/About/About.jsx [hunk 2]
```diff
@@ -3,18 +3,21 @@ import Layout from "../../components/Layout/Layout";
...
-        imageSrc="https://place-puppy.com/300x300"
-        alt="puppy"
-        description="A picture of a man's best friend"
+      <PageHeader
+        title="our story"
+        subtitle="30 Years of Breaking The Rules"
+        content="aString"
+        isVideo={true}
       />
+      <Photo imageSrc={OrchestraImg} alt="" description="-" />
```
Comment (DanForder) [inline src/pages/About/About.jsx:20]: description should be "earliest known photo of the orchestra", crop that mock image if needs be

File: src/pages/About/About.jsx [hunk 3]
```diff
@@ -3,18 +3,21 @@ import Layout from "../../components/Layout/Layout";
...
     <Layout eventId={eventId} fontType={1}>
-      <h1>About page</h1>
-      <Photo
-        imageSrc="https://place-puppy.com/300x300"
-        alt="puppy"
-        description="A picture of a man's best friend"
+      <PageHeader
+        title="our story"
+        subtitle="30 Years of Breaking The Rules"
+        content="aString"
```
Comment (DanForder) [inline src/pages/About/About.jsx:17]: use a real video here, doesn't need to be the right one, just make sure it's vaguely professional/ presentable to the client.

File: src/pages/Schedule/Schedule.jsx [hunk 4]
```diff
@@ -3,12 +3,17 @@ import { useParams } from "react-router-dom";
...
 
 const Schedule = () => {
   const { eventId } = useParams();
 
   return (
     <Layout eventId={eventId} fontType={1}>
+      <PageHeader
+        title="can you hear the shape of the drums?"
+        subtitle="Event Schedule."
+      />
```
Comment (DanForder) [inline src/pages/Schedule/Schedule.jsx:16]: this should be dynamic based on the mock data file, look at how the home page is importing the event and follow that same code

**Review by DanForder — CHANGES_REQUESTED** [review]

File: src/components/PageHeader/PageHeader.jsx [hunk 5]
```diff
@@ -8,11 +8,11 @@ const PageHeader = (props) => {
...
     props;
 
-  const headerClass =
-    (time, date, venue, location, content, isVideo) == null
-      ? "page-header-only"
-      : "page-header";
-
+  let headerClass = 'page-header'
+    if((time, date, venue, location, content, isVideo) == null){
+      headerClass +=  "--only";
```
Comment (DanForder) [inline src/components/PageHeader/PageHeader.jsx:13]: I mean to add an _extra_ modifier class, so it should be ` headerClass += " page-header--only"

File: src/components/PageHeader/PageHeader.scss [hunk 6]
```diff
@@ -3,9 +3,9 @@
...
-  background-color: $modern-primary;
-  padding-top: 5rem;
-  padding-bottom: 2rem;
-}
+
+  &--only {
+    @extend .page-header;
+    padding-bottom: 2rem;
+  }
+}
```
Comment (DanForder) [inline src/components/PageHeader/PageHeader.scss:11]: this can become just 
```
&--only{
    padding-bottom: 2rem;
}
```

**Review by DanForder — CHANGES_REQUESTED** [review]

File: src/components/PageHeader/PageHeader.scss (hunk 6 above)
Comment (noahSerafin) [reply src/components/PageHeader/PageHeader.scss:11]: this didn't work for me earlier, maybe have a go with it yourself?

Comment (noahSerafin) [reply src/components/PageHeader/PageHeader.scss:11]: oh wait i think i know what you mean now

File: src/components/PageHeader/PageHeader.jsx [hunk 7]
```diff
@@ -10,7 +10,7 @@ const PageHeader = (props) => {
 
   let headerClass = 'page-header'
     if((time, date, venue, location, content, isVideo) == null){
-      headerClass +=  "--only";
+      headerClass +=  "page-header--only";
```
Comment (DanForder) [inline src/components/PageHeader/PageHeader.jsx:13]: need a space here: `" page-header--only"`

**Review by DanForder — CHANGES_REQUESTED** [review]

File: src/components/PageHeader/PageHeader.jsx (hunk 7 above)
Comment (jasenscode) [reply src/components/PageHeader/PageHeader.jsx]: Added

**Review by DanForder — APPROVED** [review]

**Pull request merged**